interface/skin.dmf

```diff
@@ -355,12 +355,12 @@ macro "hotkeymode"
 	elem "ctrlh"
 		name = "CTRL+H"
 		command = "holster"
-	elem "j"
```

Comment:
For anyone _confused_ , this was supposed to be removed last PR. idkh it got in

Reply:
Acknowledged.